client/src/api/okteto.ts

```diff
@@ -100,10 +101,25 @@ const up = (manifestFile: string, contextName: string, onOutputChange: (stdout:
   });
 };
 
+const status = async (manifest: string, contextName: string) : Promise<OktetoStatus> => {
+  try {
+  const args = ['status', '-f', manifest, '-c', contextName]
+  const result = await window.ddClient.extension?.host?.cli.exec('okteto', args) 
+  if(result) {
+    const status = result.parseJsonObject();
+    return status.status;
```

Comment:
We should add a type guard here to make sure the status matches the type union, or return a default value otherwise.